tests/CLI/modules/vlan_tests.py

```diff
@@ -93,6 +95,163 @@ def test_vlan_edit(self, click):
+
+    def test_vlan_create_network_extra_route_exception(self):
+        _mock = self.set_mock('SoftLayer_Product_Package', 'getItems')
+        _mock.return_value = SoftLayer_Product_Package.getItemsVLAN
+        order_mock = self.set_mock('SoftLayer_Product_Order', 'placeOrder')
+        order_mock.return_value = SoftLayer_Product_Order.vlan_placeOrder
+        result = self.run_command(['vlan', 'create',
+                                   '--name', 'TEST001',
+                                   '--datacenter', 'dal09',
+                                   '--network', 'private',
+                                   '--pod', 'dal09.pod01'
+                                   ])
+        self.assertIsInstance(result.exception, exceptions.CLIAbort)
+        self.assertIn("Unable to find pod name", result.exception.message)
+
+    @mock.patch('SoftLayer.CLI.formatting.confirm')
+    def test_vlan_create_network_priv(self, confirm_mock):
+        confirm_mock.return_value = True
+        get_pods_mock = self.set_mock('SoftLayer_Network_Pod', 'getAllObjects')
+        get_pods_mock.return_value = [{
+            'name': 'ams03.pod01',
+            'frontendRouterId': 468136,
+            'backendRouterId': 468132}]
+        result = self.run_command(['vlan', 'create',
+                                   '--name', 'TEST001',
+                                   '--datacenter', 'ams03',
+                                   '--network', 'private',
+                                   '--pod', 'ams03.pod01'
+                                   ])
+
+        self.assert_called_with('SoftLayer_Network_Pod', 'getAllObjects')
+        self.assertIsInstance(result.exception, ex.SoftLayerError)
+
+    @mock.patch('SoftLayer.CLI.formatting.no_going_back')
+    def test_vlan_create_option(self, confirm_mock):
+        confirm_mock.return_value = True
+        datacenter_mock = self.set_mock('SoftLayer_Location_Datacenter', 'getDatacenters')
+        datacenter_mock.return_value = [{'id': 814994,
+                                         'name': 'ams03'
+                                         }]
+        router_mock = self.set_mock('SoftLayer_Location_Datacenter', 'getHardwareRouters')
+        router_mock.return_value = [{'hostname': 'bcr01a.ams03'}]
+        result = self.run_command(['vlan', 'create-options'])
+        print(result.output)
+        self.assert_no_fail(result)
+
+    @mock.patch('SoftLayer.CLI.formatting.no_going_back')
+    def test_vlan_list_get_pod_with_closed_announcement(self, ngb_mock):
+        ngb_mock.return_value = True
+        vlan_mock = self.set_mock('SoftLayer_Account', 'getNetworkVlans')
+        gpods_mock = self.set_mock('SoftLayer_Network_Pod', 'getAllObjects')
+        vlan_mock.return_value = {'primaryRouter': {
+            'fullyQualifiedDomainName': 'bcr01a.fra02.softlayer.com',
+            'id': 462912},
+                                  'tagReferences': ''}
```

Comment:
Fix the formatting here, tagReferences should line up with id

Reply:
Changed the formatting.